data-raw/process_data_sets.R

```diff
@@ -381,3 +381,12 @@ avocados <-
   mutate(type = as.factor(type)) %>%
   rename(xlarge_hass_sold = xlarage_hass_sold)
 usethis::use_data(avocados, overwrite = TRUE)
+
+
+# saratoga housing prices
+# original google sheet here: https://docs.google.com/spreadsheets/d/1AY5eECqNIggKpYF3kYzJQBIuuOdkiclFhbjAmY3Yc8E
+# 
+saratoga_houses <- "data-raw/saratoga_houses.csv" %>%
+  read.csv("https://wjhopper.github.io/SDS-201/data/saratoga_ny_home_prices.csv")
+usethis::use_data(saratoga_houses, overwrite = TRUE) # data file name fixed; format fixed
+
```

Comment:
`data/st_nyc_housing_prices.rda` should not be part of the PR. Please delete it and commit the deletion.

Reply:
@heschmidt can you please delete `data/st_nyc_housing_prices.rda` and commit and push the deletion?